W-19460350-Update-Maven-settings-files-and-location-CP (#485)
mulesoft/docs-code-builder: modules/ROOT/pages/_partials/acb-maven-settings.adoc

//
// tag::maven-settings-custom[]

Use the Apache Maven file `settings.xml` to configure the Maven settings, such as repositories, proxy settings, and credentials. For detailed information, see https://maven.apache.org/settings.html[Settings Reference] in the Apache Maven documentation.

> To change the default path to your Maven `settings.xml` directory, set the `maven.settings` or `M2_REPO` system property.

Reviewer: Hi @Crispy-Salesforce , Haven't seen this, but can we also remove the documentation that points to changing the settings.xml file directory.
ACB will only use the settings.xml file located at user ${user.home}/.m2/settings.xml

//
// tag::maven-acb-settings-file[]

> Anypoint Code Builder reads its configuration from your Maven `settings.xml` file. To configure Maven, for example, to add repositories, proxies, or credentials, edit your `settings.xml` file, which is typically located in your `~/.m2/` directory.

Reviewer 2: ```suggestion
Anypoint Code Builder reads its configuration from your Maven `settings.xml` file. To configure Maven, for example, to add repositories, proxies, or credentials, edit your `settings.xml` file, usually located in your `~/.m2/` directory.
```

Author: I like it! Thanks!
